src/languages/xml.js

```diff
@@ -6,10 +6,37 @@ Category: common
 
 function(hljs) {
   var XML_IDENT_RE = '[A-Za-z0-9\\._:-]+';
-    var XML_ENTITIES = {
+  var XML_ENTITIES = {
     className: 'symbol',
     begin: '&[a-z]+;|&#[0-9]+;|&#x[a-f0-9]+;'
   };
+  var XML_META_KEYWORDS = {
+	  className: 'meta-keyword',
+	  begin: '\\s#?[a-z_][a-z1-9_-]+',
+	  illegal: '\\n',
+  };
+  var XML_META_PAR_KEYWORDS = {
+	  begin: '\\(', end: '\\)',
+	  contains:[
+	    {
+	      className: 'meta-keyword',
+	      begin: '#?[a-z_][a-z1-9_-]+',
+	      illegal: '\\n',
+      }
+	  ]
+	};
+  var APOS_META_STRING_MODE = {
+	  className: 'meta-string',
+	  begin: '\'', end: '\'',
+	  illegal: '\\n',
+	  contains: [hljs.BACKSLASH_ESCAPE]
+  };
+  var QUOTE_META_STRING_MODE = {
+	  className: 'meta-string',
+	  begin: '"', end: '"',
+	  illegal: '\\n',
+	  contains: [hljs.BACKSLASH_ESCAPE]
+  };
```

Comment:
These should probably both inherit from QUOTE_STRING_MODE and APOS_STRING_MODE.  See other languages for examples.

Reply:
You're right. I did it.